pkgs/applications/misc/eureka-editor/default.nix

```
@@ -1,25 +1,22 @@
-{ lib, stdenv, fetchzip, fltk, zlib, xdg-utils, xorg, libjpeg, libGL }:
+{ lib, stdenv, fetchzip, fltk, zlib, xdg-utils, xorg, libjpeg, libGLU }:
 
 stdenv.mkDerivation rec {
   pname = "eureka-editor";
-  version = "1.21";
-  shortver = "121";
+  version = "1.27b";
+  majorVersion = "1.27";
```

```suggestion
  version = "${majorVersion}b";
  majorVersion = "1.27";
```
just to make sure not one is stale for some reason.